crates/utils/re_video/src/decode/ffmpeg_h264/ffmpeg_libloading.rs

```diff
@@ -0,0 +1,1155 @@
+//! Send video data to `ffmpeg` over CLI to decode it.
```

Comment:
Is this whole file a copy of `crates/utils/re_video/src/decode/ffmpeg_h264/ffmpeg.rs`? Did you mean to rename it but created a copy instead?

Reply:
argh. When I kept poking things on the weekend I accidentally put in this discarded experimentation file. This is not supposed to be here!